fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/AbstractInsertExecutor.java

```diff
@@ -183,6 +183,8 @@ public void executeSingleInsert(StmtExecutor executor, long jobId) throws Except
         try {
             execImpl(executor, jobId);
             if (!checkStrictMode()) {
```

Comment:
Should use on fail not on complete
checkStrictMode should throw exception. And then go to line 205 automatically.